app/models/person.rb

```diff
@@ -229,7 +229,11 @@ def meets?(requirement)
   end
 
   def upcoming_events(count = 10)
-    self.department.events.limit(count)
```

Comment:
I'd like to move this into a Setting class method. Something like 
`Setting.get_integer('Upcoming_event_count',10)`
This would look for the *active* setting. If it found one that had a non-null value, return that value. If it did not, return the default handed in (10).
The default should be optional. If there is no setting, and there is no default, return nil.
I hope to use Settings a lot in the app. I think this will be a more sustainable model. Thoughts/concerns ?

Reply:
yeah totally agree, this should be inside the Setting model, sorry it was bad decision by me.

Will update the requested changes and let you know.